src/Core/src/Handlers/View/AndroidBatchPropertyMapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Maui.Handlers
+{
+#if ANDROID
+	public class AndroidBatchPropertyMapper<TVirtualView, TViewHandler> : PropertyMapper<TVirtualView, TViewHandler>
```

Comment:
@hartez should this class be `internal`? Maybe it's not something we want developers to use?

Reply:
Yeah, it probably should at least start out `internal`. We can always make it `public` later if a good reason emerges.